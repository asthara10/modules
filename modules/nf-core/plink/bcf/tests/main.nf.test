nextflow_process {

    name "Test Process PLINK_BCF"
    script "../main.nf"
    config "./nextflow.config"
    process "PLINK_BCF"

    tag "modules"
    tag "modules_nfcore"
    tag "plink"
    tag "plink/bcf"

    test("test_plink_bcf") {


        when {
            process {
                """
                
                input[0] = [
    [ id:'test_compressed_bcf', single_end:false ], // meta map
        file(params.test_data['sarscov2']['illumina']['test_bcf'], checkIfExists: true) ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
