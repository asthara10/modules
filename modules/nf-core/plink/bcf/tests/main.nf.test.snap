{
    "versions": {
        "content": [
            [
                "versions.yml:md5,651676ad52a85d41e64b7a2fbb5027d7"
            ]
        ],
        "timestamp": "2023-11-24T14:21:54.84958"
    }
}